### PR TITLE
Issue #9146: update backport section for changes for jdk11

### PR DESCRIPTION
**Pull request by rnveach**
Issue #9146

Update documentation for backport.

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

**Comment by rnveach** [comment]
GitHub, generate website

File: src/xdocs/index.xml.vm [hunk 1]
```diff
@@ -90,17 +90,25 @@
...
-        <a href="https://github.com/rnveach/checkstyle-backport-jre6">github site</a> or the
-        <a href="https://rnveach.github.io/checkstyle-backport-jre6/">documentation site</a> on how
+        Checkstyle releases to be run with older JDKs. It is not always an easy process and provided
+        at the same release time as Checkstyle, but it tries to maintain a functional equivalent
+        alternative.
+      </p>
+      <p>
+        If you wish to continue using new Checkstyle versions on older JDKs,
+        we recommend you either checkout the
+        <a href="https://github.com/rnveach/checkstyle-backport-jre8">github site</a> or the
```
Comment (romani) [inline src/xdocs/index.xml.vm:106]: >github site

Please make it clear it is jdk8 minimal , as you mentioned jdk 1.6 project.

File: src/xdocs/index.xml.vm [hunk 2]
```diff
@@ -90,17 +90,25 @@
...
+        <a href="https://rnveach.github.io/checkstyle-backport-jre8/">documentation site</a> on how
         to use the backport version of the utility, in place of the official Checkstyle version.
+        The <a href="https://github.com/rnveach/checkstyle-backport-jre6">JDK 1.6 site</a> is
+        being deprecated because of the continued technical hurdles with porting the code to
```
Comment (romani) [inline src/xdocs/index.xml.vm:110]: It might make sense to reference latest supported port of checkstyle.

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/xdocs/index.xml.vm (hunk 1 above)
Comment (rnveach) [reply src/xdocs/index.xml.vm:106]: Done.

File: src/xdocs/index.xml.vm (hunk 2 above)
Comment (rnveach) [reply src/xdocs/index.xml.vm:110]: I may need clarification on this statement. The 1.8 min is the latest supported port, 1.6 is being deprecated. Maybe it is just a confusion in my wording. I updated this to make it a bit clearer.

**Comment by romani** [comment]
GitHub, generate website

File: src/xdocs/index.xml.vm (hunk 2 above)
Comment (romani) [reply src/xdocs/index.xml.vm:112]: >If you wish to continue using new Checkstyle versions on older JDKs

Should be some things like "If you wish to continue using new Checkstyle versions on 8  JDKs

Comment (rnveach) [reply src/xdocs/index.xml.vm:112]: Done

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
Whole section become a bit long, first paragraph is not relevant anymore. All we need is just a table of backports.
Can we do same as we did in jdk support ?

**Comment by rnveach** [comment]
I feel the first paragraph is still important, especially if we are going to document both backport versions.

Updated.

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]

**Pull request merged**